chart/templates/loadbalancer.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.externalIP -}}
```

Comment:
@samiyaakhtar We might want to add directions on how to port forward if not using a public ip.